app/components/publications/batch-documents-publication-modal.hbs

```diff
@@ -0,0 +1,51 @@
+<WebComponents::AuModal>
+  <WebComponents::AuModal::Header @title={{ t "documents" }} @closeModal={{this.onCancel}} />
+  <WebComponents::AuModal::Body>
+    {{!-- template-lint-disable table-groups --}}
+    <table class="auk-table auk-table--styled auk-table--bordered">
+      <thead>
+        <tr>
+          <th>{{t "name"}}</th>
+          <th>{{t "document-type"}}</th>
+          <th></th>
+        </tr>
+      </thead>
+
+      <Documents::DocumentList
+        @pieces={{this.pieces}}
+        {{!-- should be tbody tag; workaround for Documents::DocumentList rendering a div --}}
+        style="display: table-row-group;"
+        as |pieces|>
+        {{#each pieces as |piece|}}
+          <tr>
+            <td>
+              {{piece.name}}
+            </td>
+            <td>
+              {{piece.file.extension}}
+            </td>
+            <td>
+              <WebComponents::AuButtonToolbar>
+                <WebComponents::AuButton
+                  @layout="icon-only"
+                  @icon="download"
+                  @skin="borderless"
+                  @href="{{await piece.downloadFileLink}}"
```

Comment:
Is de `await` hier noodzakelijk?

Reply:
Gecheckt: is nodig.

Reply:
En wanneer je `@href="{{piece.file.downloadLink}}"` gebruikt? We proberen `await` zoveel mogelijk te vermijden in templates, dus zeker geen standaard maken om die altijd te zetten voor component arguments.